src/VirtoCommerce.AzureSearchModule.Data/AzureSearchProvider.cs

```diff
@@ -229,7 +229,7 @@ protected virtual Field CreateProviderField(string documentType, string fieldNam
                 IsSortable = field.IsFilterable && !isCollection,
             };
 
-            if (providerField.IsSearchable)
+            if (providerField.IsSearchable == true)
```

Comment:
Does it really need?

Reply:
Seems this check is required because of `providerField.IsSearchable`  is nullable

Reply:
Yes. It's nullable.

Reply:
OK.